queries.sql

```diff
@@ -15,3 +15,59 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3 ;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified' ;
+
+SELECT name, species from animals;
+
+ROLLBACK;
+
+SELECT name, species from animals;
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
```

Comment:
- [ Required ] I think it would be nice if you made a `SELECT` query to verify that indeed the `species` column was updated before the `COMMIT`. According to the project requirements you are supposed to verify that change was made.

## Project requirements
![Screenshot from 2023-09-28 04-14-12](https://github.com/RicardoEgas/Vet_clinic/assets/16392046/17c57945-f4a5-43c1-af6c-b7d90a4ba256)